### PR TITLE
TST: Add security audit job

### PR DESCRIPTION
**Pull request by pllim**
Follow up of #872 . This uses [bandit](https://bandit.readthedocs.io/en/latest/) package to do simple security audit. If you think this is not necessary for Ginga, we can close without merge.

### CODE REVIEW

File: ginga/gtk3w/GtkHelp.py [hunk 1]
```diff
@@ -401,9 +401,9 @@ def append_page(self, widget, label):
         # pick a random spot to place the window initially
         rect = self.get_allocation()
         wd, ht = rect.width, rect.height
-        x = random.randint(self.cascade_offset,
+        x = random.randint(self.cascade_offset,  # nosec
```
Comment (ejeschke) [inline ginga/gtk3w/GtkHelp.py:404]: Hmm, it flags use of `random` module due to seed not being truly random enough?

Comment (pllim) [reply ginga/gtk3w/GtkHelp.py:404]: No, it's actually not that relevant unless we doing crypto. See https://bandit.readthedocs.io/en/latest/blacklists/blacklist_calls.html#b311-random

**Comment by pllim** [comment]
> do you think the security audit is "comprehensive" enough to be useful?

I am not sure, but I think it doesn't hurt. For a serious security audit though, if necessary, it should be the _only_ tool, but it is a good start.

> how much time does it add to our CI?

The log says it added 27 seconds.

> Your call

I would merge it unless you have objection. We can always remove it in the future if you change your mind. Do you agree?

**Comment by ejeschke** [comment]
Yikes, 27 seconds.  We are quickly ramping back up to longer CI.  But let's go ahead and merge.  If the build becomes too long it might be one of first to get commented out.

**Pull request merged**